### PR TITLE
DTOs

### PR DESCRIPTION
**Pull request by danylo-dudok**

### CODE REVIEW

File: TransIT.DAL/Models/DTOs/IssueLogDTO.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        public int Id { get; set; }
+        public string Description { get; set; }
+        public decimal? Expenses { get; set; }
+        public StateDTO OldStateId { get; set; }
+        public StateDTO NewStateId { get; set; }
+        public SupplierDTO SupplierId { get; set; }
+        public ActionTypeDTO ActionTypeId { get; set; }
+        public IssueDTO IssueId { get; set; }
+        public UserDTO CreateId { get; set; }
+        public UserDTO ModId { get; set; }
```
Comment (CrazyRoka) [inline TransIT.DAL/Models/DTOs/IssueLogDTO.cs]: Change Property name, please

File: TransIT.DAL/Models/DTOs/IssueDTO.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public class IssueDTO
+    {
+        public int Id { get; set; }
+        public string Summary { get; set; }
+        public int? Warranty { get; set; }
+        public DateTime? Deadline { get; set; }
+        public StateDTO StateId { get; set; }
+        public int? AssignedTo { get; set; }
+        public VehicleDTO VehicleId { get; set; }
+        public MalfunctionDTO MalfunctionId { get; set; }
```
Comment (CrazyRoka) [inline TransIT.DAL/Models/DTOs/IssueDTO.cs]: Change property name

File: TransIT.DAL/Models/DTOs/TokenDTO.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace TransIT.DAL.Models.DTOs
+{
+    public class TokenDTO
+    {
+        public string RefreshToken { get; set; }
+        public string AccessToken { get; set; }
+        public DateTime? CreateDate { get; set; }
+        public DateTime? ModDate { get; set; }
```
Comment (CrazyRoka) [inline TransIT.DAL/Models/DTOs/TokenDTO.cs]: Why do we need ModDate?
We need ExpirationDate

File: TransIT.DAL/Models/DTOs/VehicleDTO.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TransIT.DAL.Models.DTOs
+{
+    public class VehicleDTO
+    {
+        public int Id { get; set; }
+        public VehicleTypeDTO VehicleTypeId { get; set; }
```
Comment (CrazyRoka) [inline TransIT.DAL/Models/DTOs/VehicleDTO.cs]: Change name

**Review by CrazyRoka — CHANGES_REQUESTED** [review]

**Review by CrazyRoka — APPROVED** [review]

**Review by IrynaPanchuk97 — APPROVED** [review]

**Pull request merged**